web-ui/src/main/resources/catalog/js/CatController.js

```diff
@@ -1225,7 +1225,7 @@
         requireProxy: [],
         gnCfg: angular.copy(defaultConfig),
         gnUrl: "",
-        docUrl: "https://geonetwork-opensource.org/manuals/4.0.x/{lang}",
+        docUrl: "https://docs.geonetwork-opensource.org/latest/{lang}",
```

Comment:
Should this have some logic based on the geonetwork version being run? 

docs.geonetwork-opensource.org will continue to list docs by <major>.<minor> version number and that would be more stable over time then "latest", "stable","maintenance" which change to reflect release cycle each year.

Reply:
For now I planned to update this value in the back ports.